stubs/tabulate/tabulate/__init__.pyi

```diff
@@ -1,15 +1,18 @@
 from collections.abc import Callable, Container, Iterable, Mapping, Sequence
 from typing import Any, NamedTuple
-from typing_extensions import TypeAlias
+from typing_extensions import Final, Self, TypeAlias
 
-__version__: str
+from .version import __version__ as __version__
 
-LATEX_ESCAPE_RULES: dict[str, str]
-MIN_PADDING: int
+__all__ = ["tabulate", "tabulate_formats", "simple_separated_format"]
+
+# These constants are meant to be configurable
```

Comment:
Not constants then are they :-)

Reply:
Hum, yeah 😅 feel free to wordsmith that.